packages/keystone/README.md

```diff
@@ -185,11 +185,21 @@ Disconnect all adapters.
 
 ## executeQuery(queryString, config)
 
-### Usage
-
 Use this method to execute queries or mutations directly against a Keystone
 instance.
 
+ℹ️ **Note:** When querying or mutating via `keystone.executeQuery`, there are differences
```

Comment:
should this be formatted like this:-
> **Note:** When querying or mutating via `keystone.executeQuery`, there are differences to keep in mind:
> 
> - No access control checks are run (everything is set to `() => true`)
> - The `context.req` object is set to `{}` (you can override this if necessary,
  see options below)
> - Attempting to authenticate will throw errors (due to `req` being mocked)

Reply:
Agree